etc/docker/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3.3"
 services:
 
   enterprise-gateway:
-    image: elyra/enterprise-gateway:${VERSION:-2.0.0rc1}
+    image: elyra/enterprise-gateway:${VERSION:-dev}
```

Comment:
Actually, this should be `elyra/enterprise-gateway:dev` and then release.sh should be updated to replace `:dev` with the release-specific tag.